src/timesync/timesyncd-manager.c

```diff
@@ -1290,3 +1449,154 @@ int bus_manager_emit_ntp_server_changed(Manager *m) {
 
         return 1;
 }
+
+#if ENABLE_TIMESYNC_NTS
+
+static int manager_nts_obtain_agreement(Manager *m) {
```

Comment:
we generally want that params are checked for basic validity, i.e. add `assert(m)` here